tools/depends/configure.ac

```diff
@@ -319,6 +340,9 @@ case $host in
         # setup which cpu to use
         if test "x$use_cpu" = "xauto"; then
           use_cpu=x86_64
+          use_cmake_cpu=x86_64
+        else
+          use_cmake_cpu=$use_cpu
```

Comment:
Not idea which can be a alternative CPU to x86_64-apple-darwin, it set here the `use_cmake_cpu` to this from `use_cpu`.

Maybe hardcode there `use_cmake_cpu=x86_64` for all?

Reply:
How would setting the custom passed cpu value instead of x86_64 be correct for `CMAKE_SYSTEM_PROCESSOR` for this host?

Reply:
Therefore this place also confused me, it doesn't really make sense to have another CPU there (about `if test "x$use_cpu" = "xauto"; then`).